pkgs/development/libraries/libpg_query/default.nix

```
@@ -13,13 +13,20 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ which ];
 
-  makeFlags = [ "build" ];
+  makeFlags = [ "build" "build_shared" ];
 
   installPhase = ''
     install -Dm644 -t $out/lib libpg_query.a
     install -Dm644 -t $out/include pg_query.h
+  '' + lib.optionalString stdenv.isLinux ''
+    install -Dm644 -t $out/lib libpg_query.so
+  '' + lib.optionalString stdenv.isDarwin ''
+    install -Dm644 -t $out/lib libpg_query.dylib
```

Sorry for the delay. I think you can use:
```suggestion
    install -Dm644 -t $out/lib libpg_query${stdenv.hostPlatform.extensions.sharedLibrary}
```
Besides that, LGTM

---

Ah, thanks, I failed to find something like this the first time around! Applied.